FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -121,6 +150,68 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
+/**
+ * Checks if prewarming is available for the platform on current device.
+ * It is available when running iOS 15 and above.
+ *
+ * @return true if the platform could prewarm apps on the current device
+ */
++ (BOOL)isPrewarmAvailable {
+  if (![[GULAppEnvironmentUtil applePlatform] isEqualToString:@"ios"]) {
```

Comment:
Curious if they have an ENUM to check for iOS. With a a string equality check, the check would fail if they change the case of the value. If there are not other options available, lets make a comparison that ignores case sensitivity.

Reply:
Will change to case insensitive compare